src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Cohost/CohostSemanticTokensRangeEndpoint.cs

```diff
@@ -55,6 +58,10 @@ public void ApplyCapabilities(VSInternalServerCapabilities serverCapabilities, V
         var clientLanguageServerManager = context.GetRequiredService<IRazorCohostClientLanguageServerManager>();
         var clientConnection = new RazorCohostClientConnection(clientLanguageServerManager);
 
-        return _semanticTokensInfoService.GetSemanticTokensAsync(clientConnection, request.TextDocument, request.Range, documentContext.AssumeNotNull(), cancellationToken);
+        // TODO: This is currently using the "VS" client settings manager, since that's where we are running. In future
```

Comment:
I assume this is why you are now injecting color background in other parts of this commit

Reply:
Yes